Composer/packages/client/src/pages/botProject/runtime-settings/RuntimeSettings.tsx

```diff
@@ -203,6 +212,16 @@ export const RuntimeSettings: React.FC<RouteComponentProps<{ projectId: string }
     );
   };
 
+  const errorElement = (errorText: string) => {
+    if (!errorText) return '';
+    return (
+      <div css={errorContainer}>
```

Comment:
I think using div here might violate DOM rules, since Fabric uses p tag for error messages. Please check dev tools for potential errors and if there are any, use span here. (or use Fabric Text component)

Reply:
it should be good now.